### PR TITLE
pulumiPackages.pulumi-language-{go,nodejs}: fix build

### PR DESCRIPTION
**Pull request by wegank**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kampka** [comment]
Technically, this compiles again, but I don't think the provides get installed correctly when trying to use it.
@wegank did you try it out and can provide some pointers ?

**Comment by wegank** [comment]
Did you include the folders containing `pulumi-language-go` (and possibly `go`) in `PATH`?

**Comment by kampka** [comment]
I'm using the `nodejs` version and it tries to download the provider from upstream which then fails

**Comment by wegank** [comment]
I couldn't reproduce the issue on my machine. Did you also have the folder containing `npm` in `PATH`?

**Comment by kampka** [comment]
Seems to work with a clean stack, so might be a problem upgrading. I don't have the time to dig into the specifics, but looks like it's good enough to merge.

File: pkgs/tools/admin/pulumi-packages/pulumi-language-nodejs.nix [hunk 1]
```diff
@@ -25,9 +20,4 @@ buildGoModule rec {
   nativeCheckInputs = [
     nodejs
   ];
-
-  postInstall = ''
-    cp nodejs/dist/pulumi-resource-pulumi-nodejs $out/bin
-    cp nodejs/dist/pulumi-analyzer-policy $out/bin
-  '';
 }
```
Comment (SuperSandro2000) [inline pkgs/tools/admin/pulumi-packages/pulumi-language-nodejs.nix:23]: meta is missing here, right?

**Pull request merged**